docs/server/installation.rst

```diff
@@ -280,6 +280,13 @@ setting appropriately in your custom settings file, ensuring that you are using
 the correct :setting:`ENGINE <DATABASE-ENGINE>` setting for your chosen
 database backend.
 
+Pootle requires `django-transaction-hooks <https://pypi.python.org/pypi/django-transaction-hooks/>`_
+to connect to the database. The following database backends are supported:
+
+  - sqlite: transaction_hooks.backends.sqlite3
+  - mysql: transaction_hooks.backends.mysql
+  - postgres: transaction_hooks.backends.postgresql_psycopg2
```

Comment:
I dont really agree

At the moment it links to the normal django db config page. If you just follow that it will create a problem so some explanation needs to be given here